.github/workflows/docker-release-build.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         working-directory: ./server
         run: npm audit --production --audit-level=critical
       - name: 📄 Codecov report upload
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v2
```

Comment:
Not related to this PR, but we might want to add the option "fail_ci_if_error: true"

<img width="878" alt="Screenshot 2021-09-16 at 10 45 36" src="https://user-images.githubusercontent.com/7365207/133541121-5f2dfee7-54a0-4ac3-968b-824b37bbe546.png">